lib/v2/screens/authentication/recover/recover_account_found/interactor/mappers/remaining_time_state_mapper.dart

```diff
@@ -14,7 +14,7 @@ const int _minuteSecond = 60;
 class RemainingTimeStateMapper {
   RecoverAccountFoundState mapResultToState(RecoverAccountFoundState currentState, int remainingTime) {
     int days = 0, hours = 0, min = 0;
-    int remainingTimeStamp = remainingTime - DateTime.now().millisecondsSinceEpoch ~/ 1000;
+    int remainingTimeStamp = (remainingTime - DateTime.now().millisecondsSinceEpoch) ~/ 1000;
```

Comment:
@gguijarro-c-chwy not forget the hierarchy of operations and the order of their evaluation 👀

Reply:
Dont tell me this is why it was counting down by two

Reply:
Yes it is.

Reply:
haha nice